### PR TITLE
backup: add br checksum concurrency

### PR DESCRIPTION
**Pull request by shuijing198799**
### What problem does this PR solve? 
Add `checksum-concurrency` filed for control the backup and restore concurrency in a node.

### Check List 

Tests 

 - E2E test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Go code change

```release-note
Add `spec.br.checksum-concurrency` filed for control the backup and restore concurrency in a node.
```

### CODE REVIEW

**Comment by DanielZhangQD** [comment]
@shuijing198799 Can `.spec.br.options` meet your requirement?

File: pkg/apis/pingcap/v1alpha1/types.go [hunk 1]
```diff
@@ -1306,6 +1306,8 @@ type BRConfig struct {
 	StatusAddr string `json:"statusAddr,omitempty"`
 	// Concurrency is the size of thread pool on each node that execute the backup task
 	Concurrency *uint32 `json:"concurrency,omitempty"`
+	// ChecksumConcurrency is the size of thread pool on each node that execute the checksum task
+	ChecksumConcurrency *uint32 `json:"checksum-concurrency,omitempty"`
```
Comment (lonng) [inline pkg/apis/pingcap/v1alpha1/types.go:1310]:
```suggestion
	ChecksumConcurrency *uint32 `json:"checksumConcurrency,omitempty"`
```

Comment (shuijing198799) [reply pkg/apis/pingcap/v1alpha1/types.go:1310]: It seems `.spec.br.options` can meet my requirement. this pr will be closed.

**Comment by shuijing198799** [comment]
> @shuijing198799 Can `.spec.br.options` meet your requirement?

It seems OK /close

**Pull request closed without merging**